components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -313,7 +328,7 @@ Updating a project begins an operation that applies all pending rule edits and t
 In this example, upon successful update, all ingested resources whose Chef Organization matches `devops` will be considered a part of the project `project-devops`.
 Only these resources will appear in Automate's dashboards when the `project-devops` project has been selected in the global project filter.
 
-As the operation takes place, you should see a percentage count up within the `Update Projects` button. 
+As the operation takes place, you should see a percentage count up within the `Update Projects` button.
```

Comment:
lol the use of the word `should` here really demonstrates our lack of confidence in the percentage

Reply:
Fixing that ;)